Library/Homebrew/tap.rb

```diff
@@ -1401,6 +1401,13 @@ def new_cask_path(token)
     cask_dir/cask_subdir/"#{token.downcase}.rb"
   end
 
+  sig { params(token: String).returns(Pathname) }
+  def new_font_path(token)
+    font_first_letter = T.must(token.split("-").second)[0].to_s
```

Comment:
Please split on `font-` instead if that's the naming format we're using. Probably worth `raise`ing if it's not in the expected format, too.